tools/per_test_timeouts.py

```diff
@@ -8,6 +8,7 @@
 Any tests not listed here will use the default timeout.
 """
 PER_TEST_TIMEOUTS = {
+    "test_articulation.py": 500,  # First test, applauncher needs longer to start
```

Comment:
Would be nicer to have a dummy script run to initialize the shaders. Right now we are making this dependent on alphanumeric ordering of the `sort` operation in the test runner.